cql3/result_set.cc

```diff
@@ -188,4 +191,80 @@ make_empty_metadata() {
     return empty_metadata_cache;
 }
 
+result_set_serialized serialize_result_set(const result_set& rs) {
```

Comment:
It does not looks like this serializes anything. It creates serializable structure. And BTW do we need this intermediate serializable structure? Is it absolutely impossible to define result_set in the idl?